### PR TITLE
Docs for host volumes for deployment containers

### PR DESCRIPTION
**Pull request by flomotlik**
To build minimal deployment containers you want to separate
building the artifact from building the deployment containers.
This gets documented here now.

### CODE REVIEW

File: _posts/docker/2015-05-25-services.md [hunk 1]
```diff
@@ -63,6 +63,46 @@ data:
...
+  image: compileimage:latest
+  volumes:
+    - ./tmp:/artifacts
+deploy:
+  build:
+    image: deploymentcontainer
+    dockerfile_path: Dockerfile.deploy
+```
+
+To create our artifact we're first running `build_artifact.sh`. To trigger the building of the deploy container we're simply running `true` so the container gets built.
```
Comment (AlexTi) [inline _posts/docker/2015-05-25-services.md]: Would add something like
> `build_artifact.sh`, which contains your artifact building instructions and is included in "compile" container

File: _posts/docker/2015-05-25-services.md [hunk 2]
```diff
@@ -63,6 +63,46 @@ data:
...
+
+```yaml
+- service: compile
+  command: "./build_artifact.sh"
+- service: deploy
+  command: true
+```
+
+In the Dockerfile for the deployment we're copying the artifact that was created in the fist step in to the container. Now we have a finished container that can be deployed to production.
```
Comment (AlexTi) [inline _posts/docker/2015-05-25-services.md]: In the Dockerfile (Dockerfile.deploy) for the deployment ...

File: _posts/docker/2015-05-25-services.md [hunk 3]
```diff
@@ -63,6 +63,46 @@ data:
...
+
+```Dockerfile
+FROM busybox
```
Comment (AlexTi) [inline _posts/docker/2015-05-25-services.md]: # Dockerfile.deploy

File: _posts/docker/2015-05-25-services.md (hunk 2 above)
Comment (flomotlik) [inline _posts/docker/2015-05-25-services.md]: Changed it to "In `Dockerfile.deploy` we're copying the ..." so its clear

File: _posts/docker/2015-05-25-services.md (hunk 3 above)
Comment (flomotlik) [inline _posts/docker/2015-05-25-services.md]: added

File: _posts/docker/2015-05-25-services.md (hunk 1 above)
Comment (flomotlik) [inline _posts/docker/2015-05-25-services.md]: Added that its part of your repository

**Comment by AlexTi** [comment]
lgtm

**Pull request merged**